DEVELOPMENT.md

```diff
@@ -0,0 +1,11 @@
+# How to Develop Proxy Wasm Cpp Host
```

Comment:
How about titling this "Development guidelines" (matching the link elsewhere) or "Developer documentation".

Reply:
SGTM.